packages/pwa-kit-dev/src/ssr/server/build-dev-server.test.js

```diff
@@ -738,59 +738,24 @@ describe('DevServer service worker', () => {
 
         test(`${name} (and handle 404s correctly)`, () => {
             const app = createApp()
+            app.get('/worker.js(.map)?', NoWebpackDevServerFactory.serveServiceWorker)
```

Comment:
This test had a bug where it didn't actually run `serveServiceWorker` on a 404, so it missed a branch in the coverage.